docs/migrations/migrations.md

```diff
@@ -0,0 +1,232 @@
+You can do that, but at your own risk!
+Only the officially pinned version are tested with each other.
+```
+
+
+### Why do we need to upgrade?
+
+If you want to use a new feature or bugfix that your version does not yet have, then you need to upgrade.
+
+That could be as simple as changing an `extends` in your buildout.
+
+```ini
+extends = https://dist.plone.org/release/5.2.6/versions.cfg
+```
+
+Would become the following.
+
+```ini
+extends =  https://dist.plone.org/release/5.2.9/versions.cfg
+```
+
+But you not only need to change the version of Plone you use, but you also need to update the database.
+
+Since a Plone site stores content and settings in its database, these may need to be modified to reflect changes in the logic or configuration of Plone.
+This is done by upgrade steps, or code that runs and updates the database to work with the new version.
+
+
+## Update, Upgrade, Migration, or Relaunch?
+
+Changing from `5.2.6` to `5.2.9` is usually not a migration, but an update or upgrade, more specifically a bugfix update.
+
+Your could use the following:
+
+Hotfix-Upgrade
+: Adding a Hotfix to Plone without changing anything else.
+
+Update
+: Bugfix version changes, for example, `5.2.6` to `5.2.9` without additional changes.
+
+Upgrade
+: Bugfix or minor version change that gives you at least one new feature.
+  For example, `5.2.6` to `5.2.9`, while adding `collective.easyform`, or even `5.0.8` to `5.1.7` when you don't change anything else.
+
+Relaunch
+: The design, content structure, or major version changes.
+
+Migration
+: The process of getting a database from one version of Plone to another version.
+
+To make it even more confusing people use *Major Update* or *Major Upgrade* when in fact they are doing a *Relaunch*.
+
+Good communication to manage expectations and imaginations is key!
+Make sure that everyone knows what you are planning to do, no matter what you call it.
+
+It can help to approach every non-trivial upgrade as a relaunch.
+It is usually much easier to scale down expectations than up.
+
+```{note}
+Arguing for a required switch of a Python 2.7 installation of Plone 5.2.6 to Python 3 (while staying on 5.2.6) is hard without taking the opportunity to change the mobile navigation.
+
+Suddenly it's a relaunch and everyone is excited.
+```
+
+
+(migrations-major-changes-label)=
+
+## Dealing with major changes in Plone
+
+In the history of Plone so far, there were three major changes:
+
+* Plone 5.0: Dexterity replaces Archetypes
+* Plone 5.2: Support for Python 3
+* Plone 6.0: Volto as the new default frontend
+
+Each of these major changes requires special treatment during an upgrade that makes upgrading much more complex.
+
+
+(migrations-plone-5.0-dexterity-replaces-archetypes-label)=
+
+### Plone 5.0: Dexterity replaces Archetypes
+
+With Plone 5.0 the default framework for content types switched from Archetypes to Dexterity.
+
```

Comment:
Missing the css/js changed in compiling/bundling here. If you mention Volto for Plone 6, require.s and the resource registries should be mentionned here as well.

Reply:
I would not mention that here since these do not required additional migration-steps.